openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -214,15 +214,15 @@ def _authenticate_first_party(request, unauthenticated_user):
         raise AuthFailedError(_('Too many failed login attempts. Try again later.'))
 
 
-def _handle_failed_authentication(user):
+def _handle_failed_authentication(user, has_authentication):
```

Comment:
Can you rename `has_authentication` to `is_user_authenticated` to clarify what it means?